schunk_lwa4d_moveit_config/launch/lwa4d_moveit_controller_manager.launch.xml

```diff
@@ -0,0 +1,6 @@
+<launch>
+<rosparam file="$(find schunk_lwa4d_moveit_config)/config/controllers.yaml"/>
+  <param name="use_controller_manager" value="false"/>
+  <param name="trajectory_execution/execution_duration_monitoring" value="false"/>
+  <param name="moveit_controller_manager" value="pr2_moveit_controller_manager/Pr2MoveItControllerManager"/>
```

Comment:
use the same controller manager as in https://github.com/ipa320/cob_manipulation/blob/indigo_dev/cob_moveit_config/launch/controller_manager.xml#L21-L22